test/common/helpers/start-server.test.js

```diff
@@ -15,7 +15,7 @@ jest.mock('hapi-pino', () => ({
   },
```

Comment:
Moving this file as an example.

The majority of repositories across Defra are following the pattern where the tests are in an independent folder outside of the application code.  Rather than storing tests alongside the code.

If we moved them then we'd be better aligning to the majority of our users.

Also we could avoid the need for the step where code files are copied to the `.server` folder without the tests.  

Having the files alongside the code, I also find restrictive as it doesn't give a natural place for my higher level functional, integration, contract tests etc.

Understand this one is a bit more tricky to implement though as the current pipelines seen to depend on the tests being somewhere in the src folder.

Reply:
I would second the motion to move tests out to a dedicated tests folder. I know that the world is split down the middle on this one, but this is the more common approach on Defra projects and I do think it keeps the codebase tidier as well as simplifying file path exclusions and things like that.

Reply:
Yeah this is one that could be a long conversation. It feels like a preference. I personally like to look at code, especially components and see all associated files. But have previously had a separate `/test` folder, mirroring the codebases structure. 

The main problem with the `/test` folder is the cognitive overload of linking things up and the refactoring of things and folder structure between the codebase and `/tests` drifting. Neither are huge problems...

Reply:
Agree it's preference.  My view is that we mirror the preference of the majority which seems to be dedicated test folder.

However, given the complexity of the switch as pipeline changes needed, perhaps a candidate for a discussion on a separate PR?